compiler/src/iree/compiler/Codegen/Common/TileDispatchUsingForall.cpp

```diff
@@ -33,6 +33,9 @@ namespace {
 struct TileAndDistributeToWorkgroupsUsingForallOpPass final
     : public impl::TileAndDistributeToWorkgroupsUsingForallOpPassBase<
           TileAndDistributeToWorkgroupsUsingForallOpPass> {
+  TileAndDistributeToWorkgroupsUsingForallOpPass(bool transposeWorkgroup) {
```

Comment:
Use the `explicit` keyword for single-argument constructors.

https://google.github.io/styleguide/cppguide.html#Implicit_Conversions